packages/create-portal-app/index.js

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env node
+
+const Listr = require('listr')
+const { program } = require('commander')
+const chalk = require('chalk')
+const prompts = require('prompts')
+const path = require('path')
+const figlet = require('figlet')
+const package = require('./package.json')
+const { copy, isPathInUse } = require('./helpers/copy')
+const { install, initGit, checkPackageVersion } = require('./helpers/install')
```

Comment:
Since `checkPackageVersion`  is unused, I think it should not be imported.

Reply:
ok, I will remove that.